src/ray/gcs/gcs_server/gcs_kv_manager.cc

```diff
@@ -141,6 +141,23 @@ void GcsInternalKVManager::HandleInternalKVKeys(
   }
 }
 
+void GcsInternalKVManager::HandleGetInternalConfig(
+    rpc::GetInternalConfigRequest request,
+    rpc::GetInternalConfigReply *reply,
+    rpc::SendReplyCallback send_reply_callback) {
+  reply->set_config(raylet_config_list_);
+  GCS_RPC_SEND_REPLY(send_reply_callback, reply, Status::OK());
+  ++counts_[CountType::GET_INTERNAL_CONFIG_REQUEST];
```

Comment:
Actually I don't think we need to record the count here. instrumented_io_context already records the total calls of `HandleGetInternalConfig`?

Reply:
done